core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala

```diff
@@ -102,7 +102,7 @@ private[ui] class ExecutorTable(stageId: Int, stageAttemptId: Int, parent: Stage
 
     listener.stageIdToData.get((stageId, stageAttemptId)) match {
       case Some(stageData: StageUIData) =>
-        stageData.executorSummary.toSeq.sortBy(_._1).map { case (k, v) =>
+        stageData.executorSummary.toSeq.sortWith(_._1.toInt < _._1.toInt).map { case (k, v) =>
```

Comment:
Can also be `sortBy(_._1.toInt)` I think. Also, are we sure these fit into 32-bit ints?